doc/build.md

```diff
@@ -55,71 +58,85 @@ Orientation does not matter. Solder based on the labeled values.
 | R2, R5 | 5.1k  |
 | R3, R4 | 75    |
 | R6     | 10k   |
-![Resistors](https://i.imgur.com/sZXMwDM.jpg)
 
-### Step 3. 1n1418 Diodes (D1-68, D71-74)
+![Resistors](https://imgur.com/haA16fU.png)
+
+### Step 3. 1n1418 Diodes (D1-68, D72-74)
 **This part has a specific orientation** - black line on the diode lines up with the square pad (points up)
-![Diodes](https://i.imgur.com/tWNR7mo.jpg)
+![Diodes](https://imgur.com/qEMczrJ.png)
+Don't forget the D72-74 ones !
+![Extra diodes](https://imgur.com/6sOD2Mj.png)
 
 ### Step 4. Capacitors (C1-4)
 Orientation does not matter. Solder based on the labeled values.
 | Ref    | Value |
 | ------ | ----- |
 | C1, C2 | 22uF  |
 | C3, C4 | 0.1uF |
-![Capacitors](https://i.imgur.com/jzcZJQy.jpg)
+
+![Capacitors](https://imgur.com/4xfcnwh.png)
 
 ### Step 5. USB Port
 Solder one of the large legs first and check that the port is flush before soldering the other three legs. \
 Solder the smaller pins by applying no-clean flux across the pins, then drag a small amount of solder across the pins until all the holes are filled.
-![USB port](https://i.imgur.com/MObHtyP.jpg)
+![USB port](https://imgur.com/2BZQcIB.png)
```

Comment:
A picture of the underside of the USB port would be helpful

Reply:
I'll take one.